packages/contracts/contracts/voting/multisig/Multisig.sol

```diff
@@ -124,10 +125,16 @@ contract Multisig is PluginUUPSUpgradeable, ProposalUpgradeable, Addresslist {
     ) public initializer {
         __PluginUUPSUpgradeable_init(_dao);
 
-        // add member addresses to the address list
         _addAddresses(_members);
 
         _updateMultisigSettings(_multisigSettings);
+
+        emit MembersAnnounced({members: _members});
```

Comment:
@novaknole, is this in agreement with the check-effects-interact pattern? 
I think events should be emitted at the bottom. 

However, I have no strong preference here, because there is no danger if we emit it earlier (as it just takes the input argument that cannot be changed by the subsequent `_updateMultisigSettings` call).

Reply:
Here you go fc2cc51492f48dc09c05d7e53be313bb9919dcc6

Reply:
Reverted as part of 541fc3a08797768dafcba7ac8372b19e48c1af71